src/components/Spin.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import {SpinProps} from "antd";
+
+const TableSpin = (loading: boolean): SpinProps => {
+    return {
+        spinning: loading,
+        delay: 1,
+        size: "large"
+    }
```

Comment:
Codacy has a fix for the issue: [Replace `····}` with `··};`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  };
```